### PR TITLE
34 Update LibraryContext.cs to use DbHelper to get connection string

### PR DESCRIPTION
**Pull request by deakgeek**
#34

### CODE REVIEW

File: content/efcore/getting-started/samples/shared/LibraryContext.cs [hunk 1]
```diff
@@ -8,6 +9,6 @@ public class LibraryContext : DbContext
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     {
-        optionsBuilder.UseSqlite("Filename=Library.db");
+        optionsBuilder.UseSqlite(ReplHelper<string>.DefaultConnectionString);
```
Comment (helmoski) [inline content/efcore/getting-started/samples/shared/LibraryContext.cs]: This isn't valid anymore.

```(C#)
using Microsoft.EntityFrameworkCore;
using System.Collections.Generic;
using REPLHelper;

public class LibraryContext : DbContext
{
    public DbSet<Book> Books { get; set; }
    public DbSet<Author> Authors { get; set; }

    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        optionsBuilder.UseSqlite(DBHelper.GetReadonlyDbConnectionString());
    }
}
```

Comment (helmoski) [reply content/efcore/getting-started/samples/shared/LibraryContext.cs]: Please test your changes using the TestPage

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/samples/shared/LibraryContext.cs (hunk 1 above)
Comment (deakgeek) [reply content/efcore/getting-started/samples/shared/LibraryContext.cs]: Do we need to remove the default connection string from the replhelper, then?

**Review by helmoski — APPROVED** [review]

**Comment by helmoski** [comment]
:shipit:

**Pull request merged**